packages/middleware-sdk-s3/src/configuration.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * All endpoint parameters with built-in bindings of AWS::S3::*
+ */
+export interface S3InputConfig {
+  /**
+   * Whether to force path style URLs for S3 objects (e.g., https://s3.amazonaws.com/<bucketName>/<key> instead of https://<bucketName>.s3.amazonaws.com/<key>
```

Comment:
nit: move doc to next line
```suggestion
   * Whether to force path style URLs for S3 objects
   * e.g., https://s3.amazonaws.com/<bucketName>/<key> instead of https://<bucketName>.s3.amazonaws.com/<key>
```